securedrop/tests/test_crypto_util.py

```diff
@@ -19,6 +19,15 @@ def setUp(self):
     def tearDown(self):
         utils.env.teardown()
 
+    def test_word_lists_imports(self):
+        """Make sure word count for each list is correct and Guard
+        against regressions regarding not stripping file contents
+        correctly.
```

Comment:
This comment should be removed or changed to just "No empty strings should appear in the wordlists."